lib/enumerize/activerecord.rb

```diff
@@ -21,8 +21,14 @@ def enumerize(name, options={})
           require 'enumerize/hooks/uniqueness'
 
           unless options[:multiple]
-            decorate_attribute_type(name, :enumerize) do |subtype|
-              Type.new(enumerized_attributes[name], subtype)
+            if method(:decorate_attribute_type).parameters.count == 2
+              decorate_attribute_type(name) do |subtype|
```

Comment:
Use `name.to_s`, it doesn't work as expected on user defined attribute unless doing that.

```suggestion
              decorate_attribute_type(name.to_s) do |subtype|
```

Reply:
Thanks! I fixed.